### ISSUE TITLE
Error with dbus similar to the previous issue

### ISSUE THREAD

**Issue by AshtakaOOf** [mjakeman/s76-scheduler-plugin#2]
The /etc/dbus-1/system.d/com.system76.Scheduler.conf has this by default inside it:
`<!DOCTYPE busconfig PUBLIC
          "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
          "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd%22%3E
<busconfig>
    <policy context="default">
        <allow send_destination="com.system76.Scheduler"/>
        <allow receive_sender="com.system76.Scheduler"/>
    </policy>
</busconfig>`
But i can't enable the service in systemctl i tried replacing the config by the text from the other issue and it worked.
I just replaced the text inside the config and i could enable the service no problem.
`<!DOCTYPE busconfig PUBLIC
          "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
          "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
<busconfig>
    <policy context="default">
        <allow send_destination="com.system76.Scheduler"/>
        <allow receive_sender="com.system76.Scheduler"/>
    </policy>
    <policy group="sudo">
        <allow send_destination="com.system76.Scheduler"/>
        <allow receive_sender="com.system76.Scheduler"/>
    </policy>
    <policy user="root">
        <allow own="com.system76.Scheduler"/>
        <allow send_destination="com.system76.Scheduler"/>
        <allow receive_sender="com.system76.Scheduler"/>
    </policy>
</busconfig>`
So just to tell this to you.

**Comment by AshtakaOOf** [comment]
Also i don't know how to formate the text on github so i'm sorry about that.

**Comment by mjakeman** [comment]
> Also i don't know how to formate the text on github so i'm sorry about that.

No worries. You can format text on github with three backticks on either side (i.e. \`\`\`).

So just to clarify, this didn't work:
```
<!DOCTYPE busconfig PUBLIC "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN" "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd%22%3E 
<busconfig>
    <policy context="default">
        <allow send_destination="com.system76.Scheduler"/>
        <allow receive_sender="com.system76.Scheduler"/>
    </policy>
</busconfig>
```

And this following dbus file did, am I correct?
```
<!DOCTYPE busconfig PUBLIC "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN" "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
<busconfig>
    <policy context="default">
        <allow send_destination="com.system76.Scheduler"/>
        <allow receive_sender="com.system76.Scheduler"/>
    </policy>
    <policy group="sudo">
        <allow send_destination="com.system76.Scheduler"/>
        <allow receive_sender="com.system76.Scheduler"/>
    </policy>
    <policy user="root">
        <allow own="com.system76.Scheduler"/>
        <allow send_destination="com.system76.Scheduler"/>
        <allow receive_sender="com.system76.Scheduler"/>
    </policy>
</busconfig>
```

**Comment by mjakeman** [comment]
So this is almost definitely an issue with how your copy of system76-scheduler is packaged. What I suspect is your version of system76-scheduler is packaged incorrectly (or maybe mine is :/) and that is causing the extension to not have the correct permissions.

Can you provide the following information please:
 - Distro
 - OS Version
 - How/where you got `system76-scheduler` from

**Comment by AshtakaOOf** [comment]
Distro: Fedora Workstation 35
And i got system76-scheduler from the copr repo you gave in the readme.md

**Comment by AshtakaOOf** [comment]
Also yes i needed to replace the .conf file with that config you gave in the other issue, and i'm too lazy to package it myself.

**Comment by mjakeman** [comment]
>  And i got system76-scheduler from the copr repo you gave in the readme.md

Ah, well that's embarrassing :sweat_smile: 

I'll have a look into it since it should have been fixed by now. I'm using it on Silverblue 35/36 without issue, but I don't imagine that should make a difference. For the mean time, I'd recommend using it with the custom config.

**Comment by KyleGospo** [comment]
Same issue here, multiple machines all running Fedora Workstation 35. The configuration change fixed all 3 of them.

**Comment by mjakeman** [comment]
This should be fixed as of the COPR package 1.2.1-1 - can't speak for other distribution methods of course.

Can you check if this is still an issue?

**Comment by mjakeman** [comment]
I think we can safely close this - the custom dbus policy seems to be working
